b2luigi/core/utils.py

```diff
@@ -229,6 +230,12 @@ def create_output_file_name(task, base_filename, result_dir=None):
         # Be sure to evaluate things relative to the current executed file, not to where we are now
         result_dir = map_folder(get_setting("result_dir", task=task, default=".", deprecated_keys=["result_path"]))
 
+    for key, value in serialized_parameters.items():
+        if isinstance(key, str) and "/" in value:
+            warnings.warn(f"Value of parameter ``{key}`` contains forward slash \"/\". "
```

Comment:
This is a generic warning, if you want I could define a new warning class, similar as you did for the `DeprecationSettingsWarning`